### PR TITLE
WIP: v4 migration

### PR DESCRIPTION
**Pull request by vagusX**
migration by antd4-codemod

Styles for LegacyForm doesnot work well, https://github.com/ant-design/compatible/issues/10

like this:

image

image

### CODE REVIEW

**Comment by chenshuai2144** [comment]
看起来改动很大呀

File: config/config.ts [hunk 1]
```diff
@@ -74,59 +72,276 @@ export default {
...
-      path: '/user',
-      component: '../layouts/UserLayout',
+      path: '/',
+      component: '../layouts/BlankLayout',
       routes: [
         {
-          name: 'login',
-          path: '/user/login',
-          component: './user/login',
+          path: '/user',
```
Comment (vagusX) [inline config/config.ts]: @chenshuai2144 貌似我把区块信息和区块代码提交上来了，貌似不太对 😂

Comment (xiaohuoni) [reply config/config.ts]: 是的

Comment (xiaohuoni) [reply config/config.ts]: 页面里面也有区块的，区块要提交到https://github.com/ant-design/pro-blocks

Comment (vagusX) [reply config/config.ts]: soga，那要重新提 mr 到 ant-design/pro-blocks 了 😂

Comment (vagusX) [reply config/config.ts]: soga，那要重新提 mr 到 ant-design/pro-blocks 了 😂

**Pull request closed without merging**